src/App.js

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
 
+const todoList = [
+  {
+    id: 1,
+    title: "Read the book Road to React",
+  },
+  {
+    id: 2,
+    title: "See videos from Learn the App",
+  },
+  {
+    id: 3,
+    title: "Submit my assigment",
+  },
+  {
+    id: 4,
+    title: "Practice JavaScript",
+  },
+];
+
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
+    <div>
```

Comment:
I'm not sure if the curriculum has covered these or not, but when you have multiple top-level elements that just need a wrapper, you can use a [React fragment](https://react.dev/reference/react/Fragment) instead of a `div`. They look like this `<>...</>`.